test/integration/utils.ts

```diff
@@ -14,7 +14,7 @@ const chai = require("chai");
 global.expect = chai.expect;
 chai.Should();
 
-export const LATEST_ARC_VERSION = "0.1.1-rc.20";
+export const LATEST_ARC_VERSION = "0.1.1-rc.21";
```

Comment:
This can be automated by fetching the version from @daostack/test-env-experimental/package.json @roienatan

Reply:
Here it supposed to be from `migration-experimental` insted of `test-env-experimental`.